cortex/primary/significant_locations.py

```diff
@@ -57,6 +57,34 @@ def _euclid(lat, lng, lat0, lng0):  # degrees -> km
         return 110.25 * ((((lat - lat0) ** 2) + (((lng - lng0) * np.cos(lat0)) ** 2)) ** 0.5)
     return _euclid(g0[0], g0[1], g1[0], g1[1])
 
+
+def remove_clusters(clusters, MAX_DIST=300):
```

Comment:
Should max_dist be passed as an optional parameter to siglocs? Correct me if I'm wrong but I think in the future we would want to pass different distances based on urban density